### PR TITLE
Page 404

### PR DESCRIPTION
**Pull request by srishtigautam21**
Added 404 page , comments for future understanding of code
, redirects  file and link of favicon
Files to check
Page404.jsx
ProductFilterFunctions.js
useDocumentTitle.js
useFilterHook.js

### CODE REVIEW

**Review by sach10-create — APPROVED** [review]
Appreciation: clean code.

File: src/component/ProductsFilter/ProductFilterFunctions.js [hunk 1]
```diff
@@ -18,6 +32,13 @@ const getRatingFilter = (categoryFilterData, filterByRatingSlider) => {
   }
 };
 
+/***
+ * This function filters products based on price
+ * @params - array of products and "high to low" or "low to high" input selected by user
```
Comment (sudipkundu999) [inline src/component/ProductsFilter/ProductFilterFunctions.js:37]: Appreciation: Good that you're using jsdoc. It makes it easy to figure out what the function in doing when someone sees this.

**Review by sudipkundu999 — APPROVED** [review]
Clean code. Good to merge.

File: src/component/ProductsFilter/ProductFilterFunctions.js (hunk 1 above)
Comment (srishtigautam21) [reply src/component/ProductsFilter/ProductFilterFunctions.js:37]: thanks sudip actually its inspired by your code only i saw it in one of your prs and implemented the same

**Pull request merged**